plugins/qcheck-stm/test/record_stm_tests.expected.ml

```diff
@@ -159,7 +159,15 @@ module Spec =
       match cmd__010_ with | Get -> Res (int, (get sut__011_))
   end
 module STMTests = (STM_sequential.Make)(Spec)
+let wrapped_init_state () =
+  let __state__012_ = Spec.init_state in
+  if true
```

Comment:
This makes me wonder: should the test deciding to generate a non-trivial `wrapped_init_state` rather be whether we have some invariants to test _after translation_, rather than `Option.is_some` at the beginning?

Reply:
Good catch.